.github/workflows/holonix-update.yaml

```diff
@@ -60,10 +60,11 @@ jobs:
           install_url: https://releases.nixos.org/nix/nix-2.24.4/install
           extra_nix_config: |
             accept-flake-config = true
-      - name: Run the Holochain bump script
+      - name: Run the Holochain and Lair bump scripts
         if: ${{ inputs.update-holochain }}
         run: |
           ./scripts/bump-holochain.sh
+          ./scripts/bump-lair.sh
```

Comment:
Same here, if we're trying to update Holochain, check if Lair needs to move at the same time